velox/expression/fuzzer/ExpressionFuzzerTest.cpp

```diff
@@ -129,10 +129,10 @@ std::unordered_set<std::string> skipFunctions = {
     "merge_sfm", // Fuzzer can generate sketches of different sizes.
     "element_at",
     "width_bucket",
-    // Fuzzer and the underlying engine are confused about TDigest functions
-    // (since TDigest is a user defined type), and tries to pass a
-    // VARBINARY (since TDigest's implementation uses an
+    // functions (since TDigest is a user defined type), and tries to pass a
```

Comment:
The beginning of the sentence appears to be lost. Please, double check.

Is there an issue about this limitation in the Fuzzer? If not, let's create one. CC: @kagamiori

Reply:
https://github.com/facebookincubator/velox/issues/14964